dashboard/dashboard-session-sso.md

```diff
@@ -149,3 +149,51 @@ TiDB Dashboard 支持基于 [OIDC](https://openid.net/connect/) 协议的单点
     ![操作示例 3](/media/dashboard/dashboard-session-sso-okta-info-3.png)
 
 至此，TiDB Dashboard 已被配置为使用 Okta 进行 SSO 登录。
+
+## 示例二：使用 Auth0 进行 TiDB Dashboard SSO 登录认证
+
+和 Okta 类似，[Auth0](https://www.okta.com/) 也可以提供 OIDC SSO 的身份认证服务。
+
+### 步骤一：配置 Auth0
+
+1. 访问 Auth0 管理后台。
+
+2. 点击左侧边栏的 **Applications** > **Applications**。
+
+3. 点击 **Create Application**，在弹出窗口中输入 Name，比如 "TiDB Dashboard"，application type 选择 "Single Page Web Application"。
+
+    ![Create Application](/media/dashboard/dashboard-session-sso-auth0-create-app.png)
+
+4. 点击 **Settings** 栏。
+
+    ![Settings](/media/dashboard/dashboard-session-sso-auth0-settings-1.png)
+
+5. **Allowed Callback URLs** 字段填写如下内容：
+
+    ```
+    http://DASHBOARD_IP:PORT/dashboard/?sso_callback=1
+    ```
+
+    以上内容中，将 `DASHBOARD_IP:PORT` 替换为你在浏览器中实际访问 TiDB Dashboard 所使用的域名（或 IP）及端口。
+
+    此字段可以填入多个值，用逗号分隔，用于不同的环境，比如用于开发环境则可以加上 `http://localhost:3001/dashboard/?sso_callback=1`。
+
+6. **Allowed Logout URLs** 字段填写如下内容：
+
+    ```
+    http://DASHBOARD_IP:PORT/dashboard/
+    ```
+
+    类似地，将 `DASHBOARD_IP:PORT` 替换为实际的域名（或 IP）及端口。以及支持填入多个值，用于开发环境则可以加上 `http://localhost:3001/dashboard/` (此时，开发环境下访问 TiDB Dashboard 前端首页要使用 `http://localhost:3001/dashboard/`，而不是 `http://localhost:3001/`)。
+
+    ![Settings](/media/dashboard/dashboard-session-sso-auth0-settings-2.png)
+
+7. 其它设置保持默认，点击 **Save Changes** 保存。
+
+### 步骤二：获取 TiDB Dashboard 所需的配置参数并填入 TiDB Dashboard
+
+1. 将 Auth0 **Settings** 栏 **Basic Information** 项的 **Client ID** 字段的值填入 TiDB Dashboard 的 **OIDC Client ID**，将 **Domain** 字段的值，加上 `https://` 前缀和 `/` 后缀后填入 **OIDC Discovery URL** 中。完成授权并保存配置即可。
```

Comment:
I guess the `/` suffix is optional. I'm not sure. You may have a try :)

Reply:
I have tried, miss the `/` will report an error, maybe we can add it automatically?

Reply:
Yep, auth0 needs but okta doesn't need.

Reply:
It's a wired behaviour. Maybe we should revisit the implementation.

Reply:
ok, let me check